client/signup/steps/starting-point/index.tsx

```diff
@@ -17,13 +17,15 @@ interface Props {
 	initialContext: any;
 }
 
-const EXCLUDE_STEPS: { [ key in StartingPointFlag ]: string[] } = {
+const EXCLUDE_STEPS: { [ key: string ]: string[] } = {
```

Comment:
BTW, super nit, non-blocking, can be later or not at all: grammatically the noun form of this should be "excluded steps", not "exclude steps". So `EXCLUDED_STEPS`, `getExcludedSteps()` etc. I was a bit confused reading the logic initially because of this. 😄

Reply:
Thanks for correcting me 👍